opengrok-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -47,6 +47,16 @@
             <param-value>max-age=86400</param-value>
         </init-param>
     </filter>
+    <filter>
+        <filter-name>CorsFilter</filter-name>
+        <filter-class>org.opengrok.web.CorsFilter</filter-class>
+    </filter>
+    <filter-mapping>
+        <filter-name>CorsFilter</filter-name>
+        <url-pattern>/search</url-pattern>
+        <url-pattern>/suggest</url-pattern>
+        <url-pattern>/suggest/config</url-pattern>
```

Comment:
it shouldn't be here (see my general comment on this PR) and the paths are `/api/v1/search` and so on.